yaml_files/simphony_metadata.yml

```diff
@@ -503,32 +496,21 @@ CUDS_KEYS:
 
   INLET_OUTLET:
     definition: Inlet outlet boundary condition (outlet condition is zero gradient and inlet given variable value)
-    parent: CUBA.CONDITION
+    parent: CUBA.VALUE_CONDITION
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.MATERIAL:
-      shape: (:)
-      default: []
 
   WETTING_ANGLE:
-    parent: CUBA.CONDITION
+    parent: CUBA.VALUE_CONDITION
     definition: Volume fraction wall boundary condition with specified contact angle
     models: [CUBA.CONTINUUM]
-    CUBA.VARIABLE:
-      shape: (:)
-      default: []
-    CUBA.CONTACT_ANGLE:
-      default: 45.0
+    variable: CUBA.VOLUME_FRACTION
```

Comment:
If you want variable properties to be readonly, then the distinction between fixed and variable properties is broken and we should have cuba vs. non-cuba properties. In that case, I need to be able to specify a cuba property as readonly, something I cannot currently do. This will mean changing the specs, changing the file and changing the parser.